src/app/CommandHandler.h

```diff
@@ -59,6 +59,55 @@ class CommandHandler : public Command
         virtual void OnDone(CommandHandler * apCommandObj) = 0;
     };
 
+    class Handle
+    {
+    public:
+        Handle() {}
+        Handle(const Handle & handle) = delete;
+        Handle(Handle && handle)
+        {
+            mpHandler        = handle.mpHandler;
+            handle.mpHandler = nullptr;
+        }
+        Handle(decltype(nullptr)) { Release(); }
```

Comment:
Why do we need to `Release()` here?  We're in the constructor; we know `mpHandler` is nullptr here.